src/Security/UserProvider/CustomerObjectUserProvider.php

```diff
@@ -74,26 +76,32 @@ public function loadUserByUsername($username)
 
     /**
      * @inheritdoc
+     *
+     * @return UserInterface
      */
-    public function refreshUser(UserInterface $user)
+    public function refreshUser(UserInterface $user)//: UserInterface
```

Comment:
```suggestion
    public function refreshUser(UserInterface $user)
```
Return type hint was not added on Symfony 6

Reply:
But shouldn't we add it anyway? It could come in version 7.

Reply:
I avoid this adding now so that we don't accidently enable it for Symfony 6.